frappe/website/doctype/web_page/web_page.json

```diff
@@ -289,6 +284,27 @@
    "fieldname": "meta_image",
    "fieldtype": "Attach Image",
    "label": "Image"
+  },
+  {
+   "default": "0",
+   "description": "Map route parameters into form variables. Example <code>/project/&lt;name&gt;</code>",
```

Comment:
Can we hide this check if server scripts is not enabled?

Reply:
Dynamic Jinja templates are not dependent on server scripts

Reply:
Yeah, you're right, it's not explicitly dependent, however, (correct me if I'm wrong) the dynamic route is not quite usable without writing the context script, the user has to at least write the `get_doc` for it.

`safe_exec` is blocked outright if server_scripts are not enabled. This is run when building context `safe_exec(self.context_script, None, _locals)`

https://github.com/frappe/frappe/blob/fdf74f131f2dc60d1a3b99412a2806d9c708d056/frappe/utils/safe_exec.py#L18-L21

Reply:
You can use get_doc inside Jinja without server script

Reply:
ohh! That's neat!!! I think we're good to merge!